config/test.exs

```diff
@@ -1,6 +1,8 @@
 import Config
 
 config :orbit,
+  appcues_enabled?: true,
+  appcues_id: "APPKEWES",
```

Comment:
Will this send data to appcues when running end to end tests with a headless browser?

Reply:
Ooh, possibly! Changed to use `reassign_env` as we already did for sentry. 06f21df